test/wdio/scoped-basic/cmp.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { h, Fragment } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+import exp from 'constants';
```

Comment:
```suggestion
```

Reply:
oh I didn't even see that! wonder why it was there 🤔